src/wkt/Cargo.toml

```
@@ -24,17 +24,19 @@ keywords.workspace   = true
 categories.workspace = true
 
 [features]
+chrono = []
 time = []
 
 [dependencies]
 serde      = { version = "1.0.214", features = ["serde_derive"] }
 serde_with = { version = "3.11.0", default-features = false, features = ["base64"] }
 serde_json = "1.0.134"
 time       = { version = "0.3.36", features = ["formatting", "parsing"] }
+chrono     = { version = "0.4.39" }
```

I think you can say

```suggestion
chrono     = { version = "0.4.39", optional = true }
```

and in the features section:

```
chrono = ["dep:chrono"]
```

---

Sounds good, made chrono an optional dependency.